drivers/modem/modem_receiver.c

```diff
@@ -193,6 +193,25 @@ int mdm_receiver_send(struct mdm_receiver_context *ctx,
 	return 0;
 }
 
+int mdm_receiver_sleep(struct mdm_receiver_context *ctx)
+{
+	uart_irq_rx_disable(ctx->uart_dev);
+#ifdef DEVICE_PM_LOW_POWER_STATE
```

Comment:
Question: where does this get defined? (DEVICE_PM_LOW_POWER_STATE)

Reply:
sorry for the late reply... 
DEVICE_PM_LOW_POWER_STATE is part of CONFIG_DEVICE_POWER_MANAGEMENT